custom_components/powercalc/data/emos/ZQZ832R/model.json

```
@@ -0,0 +1,13 @@
+{
+    "calculation_strategy": "lut",
+    "measure_description": "Measured with utils/measure script",
+    "measure_device": "Tapo P110",
+    "measure_method": "script",
+    "measure_settings": {
+        "SAMPLE_COUNT": 2,
+        "SLEEP_TIME": 3,
+        "VERSION": "v1.10.6:docker"
+    },
+    "name": "EMOS ZQZ832R",
+    "standby_power": 0,24
```

```suggestion
    "standby_power": 0.24
```

Syntax error in the JSON. must be period instead of comma.